### PR TITLE
ui fixes - compare view and archieved repo issues

### PR DESCRIPTION
**Pull request by Cherrg**
* add 'repo archived' message on archived repos issues (Img 1)
* add navbar to compare view (second commit) (Img 2)
* change headline on compare view of archived repos (Img 2)
* add 'repo archived' message  on compare view of archived repos (Img 2)

Signed-off-by: Michael Gnehr <michael@gnehr.de>

## Images
* **Red boxes are added**
* **red line is changed**

grafik

grafik

### CODE REVIEW

**Comment by lunny** [comment]
@Cherrg I don't think it fixed #7304, it still has a line that should not be displayed.

image

File: templates/repo/header.tmpl [hunk 1]
```diff
@@ -46,7 +46,7 @@
 		</div><!-- end grid -->
 	</div><!-- end container -->
 {{end}}
-{{if not .IsDiffCompare}}
```
Comment (zeripath) [inline templates/repo/header.tmpl:49]: Why did you remove this check?

Comment (Cherrg) [reply templates/repo/header.tmpl:49]: second image above (in merge description) -> will add first red box

These links are missing on `compare` view

File: templates/repo/diff/compare.tmpl [hunk 2]
```diff
@@ -60,7 +64,15 @@
         	<div class="ui info message show-form-container">
         		<button class="ui button green show-form">{{.i18n.Tr "repo.pulls.new"}}</button>
         	</div>
-        	{{end}}
+        	{{ else }}
+        		<div class="ui warning message">
+        			{{if .Issue.IsPull}}
+        				{{.i18n.Tr "repo.archive.title"}}
+        			{{else}}
+        				{{.i18n.Tr "repo.archive.title"}}
```
Comment (lunny) [inline templates/repo/diff/compare.tmpl]: It seems the two titles are the same?

**Review by lunny — CHANGES_REQUESTED** [review]

File: templates/repo/diff/compare.tmpl (hunk 2 above)
Comment (Cherrg) [reply templates/repo/diff/compare.tmpl]: you right, this does not belong there, missed that, sorry

Comment (Cherrg) [reply templates/repo/diff/compare.tmpl]: should be fixed

**Review by lunny — APPROVED** [review]

**Review by lafriks — APPROVED** [review]
